### PR TITLE
TLC-340 Added tests for Registration page 2(negative)

### PR DESCRIPTION
**Pull request by NadiaDziganchuk**

### CODE REVIEW

File: 20605/testData/register.js [hunk 1]
```diff
@@ -13,12 +13,22 @@ const negativeRegisterStep1 = {
...
+    email: 'Asdfgh@mail.us',
+    password: 'Azxcvb@123'
 }
 
-export {newUser, inputRegisterStep1, negativeRegisterStep1}
+const negativeRegisterStep2 = {
+    invalidPhoneNumberMin: `${'5'.repeat(9)}`,
+    invalidPhoneNumberMax: `${'5'.repeat(12)}`,
+    symbolsInvalidPhoneNumber: `${'#'.repeat(10)}`,
+    lettersUppInvalidPhoneNumber: `${'C'.repeat(9)}`,
```
Comment (aliakseyevavikt) [inline 20605/testData/register.js:28]: repeat(9) => repeat(10) or 11.

Comment (NadiaDziganchuk) [reply 20605/testData/register.js:28]: PhoneNumber accepts 10 or 11 numbers

Comment (NadiaDziganchuk) [reply 20605/testData/register.js:28]: Thanks, change repeat 10

File: 20605/specs/regression/registerStep2Negative.spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    it('should Phone Number field can not accept symbols and an error message display', function () {
+        RegisterStep2Page.phoneNumber.setValue(negativeRegisterStep2.symbolsInvalidPhoneNumber);
+        RegisterStep2Page.errorMessages[0].waitForDisplayed({timeout: WAIT_TIME_MEDIUM});
+        expect(RegisterStep2Page.errorMessages[0].getText()).eq(errorMessagesRegisterStep2.invalidPhoneNumberErrorMessage2);
+    });
+
+    it('should Phone Number field can not accept Upper letters and an error message display', function () {
+        RegisterStep2Page.phoneNumber.setValue(negativeRegisterStep2.lettersUppInvalidPhoneNumber);
+        RegisterStep2Page.errorMessages[0].waitForDisplayed({timeout: WAIT_TIME_MEDIUM});
+        expect(RegisterStep2Page.errorMessages[0].getText()).eq(errorMessagesRegisterStep2.invalidPhoneNumberErrorMessage2 + '\n' + errorMessagesRegisterStep2.invalidPhoneNumErrorMessage1);
```
Comment (Edemina) [inline 20605/specs/regression/registerStep2Negative.spec.js:32]: This it fails:
"+ '\n' + errorMessagesRegisterStep2.invalidPhoneNumErrorMessage1"    should be removed

Comment (NadiaDziganchuk) [reply 20605/specs/regression/registerStep2Negative.spec.js:32]: Changed lettersUppInvalidPhoneNumber => repeat(12) => I check both error messages.

Comment (Edemina) [reply 20605/specs/regression/registerStep2Negative.spec.js:32]: Looks good now!

**Pull request merged**